Foundation/Summit/2024/program.html

```diff
@@ -0,0 +1,373 @@
+---
+#    Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+#    SPDX-License-Identifier: CC-BY-SA-4.0
+title: Summit program
+---
+
+<h1>
+  seL4 Summit 2024
+</h1>
+<h4>
+  Sydney, Australia
+</h4>
+<h4>
+  15-17 October 2024
+</h4>
+
+<h3>
+   Program
+</h3>
+<p>
+  All times are local time in Sydney, Australia (GMT+11).
+  <!--br>
+  Session chairs in <em>italics</em>.-->
+</p>
+<table id="summit-talks">
```

Comment:
I haven't had time to check all titles/times/authors. If you'd like another pair of eyes, please ask Ihor or Nick.